chapters/extensions/VK_KHR_synchronization2.md

```diff
@@ -1,22 +1,24 @@
 # VK_KHR_synchronization2
 
-The following information was written with the goal of helping guide developers wanting to update to using `VK_KHR_synchronization2` where possible. It will also give some examples of how the original Vulkan synchronization is different.
+The `VK_KHR_synchronization2` extension provides improvements to pipeline barriers, events, image layout transitions and queue submission. This document shows the difference between the original Vulkan synchronization operations and those provided by the extension. There are also examples of how to update application code to make use of the extension.
 
 # Rethinking Pipeline Stages and Access Flags
 
 One main change with the extension is to have pipeline stages and access flags now specified together in memory barrier structures. This makes the connection between the two more obvious.
 
-The only new type of structure needed is the `VkDependencyInfoKHR` struct which wraps all the barriers into a single location.
+The only new type of structure needed is `VkDependencyInfoKHR`, which wraps all the barriers into a single location.
 
 ![VK_KHR_synchronization2_stage_access](images/VK_KHR_synchronization2_stage_access.png)
 
 ## Adding barriers for setting events
 
-Note that with the introduction of `VkDependencyInfoKHR` that `vkCmdSetEvent2KHR`, unlike `vkCmdSetEvent`, has the ability to add barriers. This was added to allow the `VkEvent` to be more useful.
+Note that with the introduction of `VkDependencyInfoKHR` that `vkCmdSetEvent2KHR`, unlike `vkCmdSetEvent`, has the ability to add barriers. This was added to allow the `VkEvent` to be more useful. Because the implementation of a synchronization2 `VkEvent` is likely to be substantially different from a Vulkan 1.2 `VkEvent`, you must not mix extension and core api calls for a single VkEvent. For example, you must not call `vkCmdSetEvent2KHR()` and then `vkCmdWaitEvents()`.
```

Comment:
```suggestion
Note that with the introduction of `VkDependencyInfoKHR` that `vkCmdSetEvent2KHR`, unlike `vkCmdSetEvent`, has the ability to add barriers. This was added to allow the `VkEvent` to be more useful. Because the implementation of a synchronization2 `VkEvent` is likely to be substantially different from a Vulkan 1.2 `VkEvent`, you must not mix extension and core api calls for a single `VkEvent`. For example, you must not call `vkCmdSetEvent2KHR()` and then `vkCmdWaitEvents()`.
```